tests/integration/targets/eos_bgp_address_family/templates/populate.cfg

```diff
@@ -0,0 +1,24 @@
+router bgp 65535
+   neighbor n2 peer-group
+   neighbor n2 next-hop-unchanged
+   neighbor n2 maximum-routes 12000 
+   neighbor peer2 peer-group
+   neighbor peer2 maximum-routes 12000 
+   network 1.1.1.0/24
```

Comment:
The tests seem to use IP addresses and BGP ASNs that fall outside the reserved/documentation range. This might be an issue for integration tests since we run these against an actual appliance.

Reply:
Ref: https://tools.ietf.org/html/rfc5398 and https://tools.ietf.org/html/rfc5737